### PR TITLE
Add config option for skipping hostname resolution in Gelf logging

### PR DESCRIPTION
**Pull request by geoand**
Relates to: #25643

### CODE REVIEW

File: extensions/logging-gelf/runtime/src/main/java/io/quarkus/logging/gelf/GelfLogHandlerRecorder.java [hunk 1]
```diff
@@ -15,7 +17,18 @@ public RuntimeValue<Optional<Handler>> initializeHandler(final GelfConfig config
...
         }
 
+        String previousSkipHostnameResolution = null;
+        if (config.skipHostnameResolution) {
+            previousSkipHostnameResolution = System.setProperty(PROPERTY_LOGSTASH_GELF_SKIP_HOSTNAME_RESOLUTION, "true");
+        }
         final JBoss7GelfLogHandler handler = new JBoss7GelfLogHandler();
+        if (config.skipHostnameResolution) {
+            if (previousSkipHostnameResolution == null) {
+                System.clearProperty(PROPERTY_LOGSTASH_GELF_SKIP_HOSTNAME_RESOLUTION);
```
[the remark this answers is not in the document]
Comment (geoand) [reply extensions/logging-gelf/runtime/src/main/java/io/quarkus/logging/gelf/GelfLogHandlerRecorder.java:27]: Not true

[the remark this answers is not in the document]
Comment (yazinnnn) [reply extensions/logging-gelf/runtime/src/main/java/io/quarkus/logging/gelf/GelfLogHandlerRecorder.java:27]: ok, it works anyway.

**Comment by geoand** [comment]
> Why do you set back the system property ? If it's only used by the logging-gelf library it seems pointless

The idea is that we should never taint the environment unless it's necessary.

> I don't really understand the impact of setting originHost and what it does ? Can this be a secrity risk ?

Not sure, but why do you think it's a security issue? FWIW, none of this new stuff is enabled by default.

**Review by loicmathieu — APPROVED** [review]
OK for me, your answers make sense

**Pull request merged**